pkgs/tools/misc/gammy/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, qmake, libXxf86vm, wrapQtAppsHook, ... }:
+
+let
+  pname = "gammy";
+  version = "0.9.58";
+in
+
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "Fushko";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "02f19b7acrzip4kbfjgqk06xv1c257rq77khpdg5gz0ai6ayvwm8";
+  };
+
+  nativeBuildInputs = [ qmake wrapQtAppsHook ];
+
+  buildInputs = [ libXxf86vm ];
+
+  installPhase = ''
+    install gammy -Dt $out/bin/
```

Comment:
When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`.

If you are up to it, you could also make a pull request against the project to make the [installation path in the project build file](https://github.com/Fushko/gammy/blob/65fbbb203d4d9b196325ef2fefc23294e944bd8f/Gammy.pro#L58) configurable like here: https://github.com/corrados/jamulus/pull/247/files

Reply:
Alternately, it would be nice to open an issue upstream asking to have the qmake file install the executable into prefix and add a comment linking there so that we know when to remove this line.

Reply:
Good idea, done.